server/services/sessionService.ts

```diff
@@ -11,15 +12,19 @@ export default class SessionService {
 
   setupSession = async () => {
     try {
-      this.request.session.handover = await this.handoverContextService.getContext(this.getToken())
+      this.request.session.handover = await this.handoverContextService.getContext(this.request.user?.token)
       this.request.session.plan = await this.planService.getPlanByOasysAssessmentPk(this.getOasysAssessmentPk())
     } catch (e) {
       Logger.error('Failed to setup session:', e)
       throw Error(e)
     }
   }
 
-  private getToken = () => this.request.user?.token
+  getToken = () => this.request.session.token
+
+  setToken = (token: Token) => {
+    this.request.session.token = token
+  }
```

Comment:
This could probably just be moved into hmppsAuthClient, as it's super unlikely we'll need to access this token elsewhere.

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/136/commits/5fb51a21fa85dc9f0b8cc486751e895965630443